admin/commands/pmm/server/install.go

```diff
@@ -0,0 +1,35 @@
+// Copyright 2019 Percona LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package server
+
+import "github.com/percona/pmm/admin/commands"
+
+// InstallCommand is used by Kong for CLI flags and commands.
+type InstallCommand struct{}
+
+type installResult struct{}
+
+// Result is a command run result.
+func (res *installResult) Result() {}
+
+// String stringifies command result.
+func (res *installResult) String() string {
+	return "works"
```

Comment:
is this code just boilerplate and will something more be added later?

Reply:
Yes, this would be just a boilerplate to be implemented later. This is to illustrate how commands would work and to have at least one command available in the new binary.